lib/commands/cdn/lib/aws-credentials.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+'use strict'
+
+const ora = require('ora')
+const request = require('request')
+
+function awsCredentials(
+  cfg /* : Object */,
+  env /* : string */,
+  accessToken /* : string | void */,
+) /* : Promise<Object> */ {
+  const spinner = ora({ spinner: 'dots', text: 'Authenticating...' }).start()
+  return new Promise((resolve, reject) => {
+    request(
+      `${cfg.service.origin}/v1/service-instances/${cfg.scope}/environments/${env}/aws-role`,
```

Comment:
This should be coming from the tenant object.

Reply:
Oh, so we aren't overriding the origin in blinkmrc anymore? just env var?